draft-ietf-sedate-datetime-extended.md

```diff
@@ -179,26 +179,27 @@ semantic result is no longer a fixed timestamp that is referenced to a
 For instance, it does not address:
 
 * Future time given as a local time in some specified time zone, where
-  changes to the definition of that time zone (e.g., a political
+  changes to the definition of that time zone (such as a political
   decision to enact or rescind daylight saving time) affect the
   instant in time corresponding with the timestamp.
 * "Floating time", i.e., a local time without information describing
   the UTC offset or time zone in which it should be interpreted.
-* The use of timescales different from UTC, such as TAI.
+* The use of timescales different from UTC, such as International Atomic
+  Time (TAI).
 
 However, additional information augmenting a fixed timestamp may be
 sufficient to detect an inconsistency between intention and the actual
-information in the timestamp, e.g., between the UTC offset and time zone
-name in the timestamp.
-For instance, such an inconsistency might arise because of:
+information in the timestamp, such as between the UTC offset and time zone
+name.
+For instance, such inconsistencies might arise because of:
 
 * political decisions as discussed above, or
 * updates to time zone definitions being applied at different times
   by timestamp producers and receivers, or
-* errors in the applications producing and consuming such a timestamp.
+* errors in the applications producing and consuming such timestamps.
 
-While the information available is not generally sufficient to resolve
-the inconsistency, it may be used to initiate some out of band
+While the information available in an IXDTF string is not generally sufficient to resolve
```

Comment:
```suggestion
While the information available in an IXDTF string is usually insufficient to resolve
```

Reply:
I like the somewhat weaker statement in the present text.